pennylane/workflow/__init__.py

```diff
@@ -53,6 +53,8 @@
     ~workflow.jacobian_products.DeviceJacobianProducts
     ~workflow.jacobian_products.LightningVJPs
 
+.. include: return_types_spec.rst
```

Comment:
Are we expecting this to appear here: https://xanaduai-pennylane--5418.com.readthedocs.build/en/5418/code/qml_workflow.html?

Reply:
Think I missed an extra colon.  Will push that change with my next draft 🤞